src/BenchmarkDotNet/Portability/Cpu/ProcCpuInfoProvider.cs

```diff
@@ -18,9 +19,28 @@ private static CpuInfo Load()
             if (RuntimeInformation.IsLinux())
             {
                 string content = ProcessHelper.RunAndReadOutput("cat", "/proc/cpuinfo");
+                var output = GetCpuSpeed();
+                content = content + output;
                 return ProcCpuInfoParser.ParseOutput(content);
             }
             return null;
         }
+        
+        private static string GetCpuSpeed()
+        {
+            var output = ProcessHelper.RunAndReadOutput("/bin/bash", "-c \"lscpu | grep MHz\"")?
+                                      .Split('\n')
+                                      .SelectMany(x => x.Split(':'))
+                                      .ToArray();
+            
+            if (output == null || output.Length < 6)
+                return null;
+                
+            double.TryParse(output[1].Trim(), out double currentValue);
+            double.TryParse(output[3].Trim().Replace(',', '.'), out double maxValue);
+            double.TryParse(output[5].Trim().Replace(',', '.'), out double minValue);
```

Comment:
It would be nice to add a comment with an example of the output. It will help to understand what kind of data do we parse.